sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BatchLoads.java

```diff
@@ -485,7 +485,7 @@ public WriteResult expandUntriggered(PCollection<KV<DestinationT, ElementT>> inp
     // loading.
     PCollectionTuple partitions =
         results
-            .apply("ReifyResults", new ReifyAsIterable<>())
+            .apply("ReifyResults", new CombineAsIterable<>())
```

Comment:
This won't have any output for an empty PCollection (whereas ReifyAsIterable does).

Reply:
yes that's right, and it fails a unit test (changed below).

I'm not fully understand the implication of this behavior of change. One thing I could wonder is that if a pipeline didn't have any input before, it would still create an empty table; while now it does not.

Could this be a problem?

Reply:
I think this could be a problem. (I'm also concerned because the underlying issue is still there...)

Reply:
The customer is waiting for this workaround. @Abacn possible to add one option to enable this workaround?

Reply:
It's now guarded by a pipeline option "--groupFilesFileLoad". PTAL, thanks!